### ISSUE TITLE
Add GRF processing capabilities to the asset server

### ISSUE THREAD

**Issue by Duckwhale** [RagnarokResearchLab/RagLite#8]
Goals:

- [x] Should be able to transparently serve files from within GRF archives
- [x] The archive should be opened at launch and closed on shutdown (decode metadata only once)
- [x] Responses should use content type `application/octet-stream` only (for now), even if the file type is known
- [x] Requests for files that do not exist should prompt a 404 response (not a crash)
- [x] Can use a custom GRF for testing and CI runs
- [x] Should support GRF 2.0 only (for now)
- [ ] Can translate virtual paths (REST routes to actual file names) - separate issue?
